packages/core/src/actor.ts

```diff
@@ -83,63 +77,6 @@ export function fromMachine<
   return new ObservableActorRef(createMachineBehavior(machine, options), name);
 }
 
-export function spawn<TReceived extends EventObject, TEmitted>(
-  behavior: Behavior<TReceived, TEmitted>,
-  // TODO: use more universal uniqueid)
-  name: string = registry.bookId()
-) {
-  const actorRef = new ObservableActorRef(behavior, name);
-  return capturedState.captureSpawn(actorRef, name);
-}
-
-export function spawnPromise<T>(
-  lazyPromise: Lazy<PromiseLike<T>>,
-  name?: string
-) {
-  return spawn(createPromiseBehavior(lazyPromise), name);
-}
-
-export function spawnObservable<T extends EventObject>(
-  lazyObservable: Lazy<Subscribable<T>>,
-  name?: string
-) {
-  return spawn(createObservableBehavior(lazyObservable), name);
-}
-
-export function spawnMachine(machine: AnyStateMachine, name?: string) {
-  return spawn(createMachineBehavior(machine), name);
-}
-
-export function spawnCallback(callback: InvokeCallback, name?: string) {
-  return spawn(
-    createDeferredBehavior(() => callback),
-    name
-  );
-}
-
-export function spawnFrom<TEvent extends EventObject, TEmitted>(
```

Comment:
This comment applies to `createBehaviorFrom` but that isn't displayed anywhere in the diff so I'm just using this location for this comment :P

We need the argument to be lazy, some entity~ types are lazy by default, others - not so much. To make non-lazy entity lazy we need to wrap it in a factory function, like here:
```js
spawnFrom(() => promise)
```
The problem with this approach is that we can't introspect the final "type" in the `spawnFrom` because we can't invoke this function and check it.

So I'm unsure how we can provide a `*From` API for this.

Reply:
We can omit this for now.